public/components/overview/mitre_attack_intelligence/resource_detail_references_table.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Wazuh app - React component for showing the Mitre Att&ck intelligence flyout tables.
+ *
+ * Copyright (C) 2015-2021 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+
+import React , {useState, useEffect} from 'react';
+import { WzRequest } from '../../../react-services';
+import {
+  EuiBasicTableColumn,
+  EuiAccordion,
+  SortDirection,
+  EuiInMemoryTable,
+} from '@elastic/eui';
+import { Markdown } from '../../common/util/markdown';
+
+type tablePropsType = (item) => {onClick: () => void};
+type backToTopType = () => void;
+
+interface referencesTableType {
+  referencesName: string,
+  referencesArray: Array<string>,
+  tableProps: tablePropsType,
+  backToTop: backToTopType
+}
+
+export const ReferencesTable = ({referencesName, referencesArray, tableProps, backToTop} : referencesTableType) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [data, setData] = useState<any[]>([]);
+  useEffect(() => {
+    getValues();
+    backToTop();
+  }, [referencesArray]);
+
+  const getValues = async () => {
+    setIsLoading(true);
+    // We extract the ids from the references tables and count them in a string for the call that will extract the info
+    const maxLength = 8100;
+    const namesConcatenated = referencesArray.reduce((namesArray = [''], element) => {
+      namesArray[namesArray.length -1].length >= maxLength && namesArray.push('');
+      namesArray[namesArray.length -1] += `${namesArray[namesArray.length -1].length > 0 ? ',' :''}${element}`;
+      return namesArray;
+    }, ['']);
+
+    // We make the call to extract the necessary information from the references tables
+    try{
+      const data = await Promise.all(namesConcatenated.map(async (nameConcatenated) => {
+        const queryResult = await WzRequest.apiReq('GET', `/mitre/${referencesName}?${referencesName.slice(-1) === 's' ? referencesName.substring(0, referencesName.length-1) : referencesName}_ids=${nameConcatenated}`, {});
```

Comment:
you could simplyfy this 
```js
${referencesName.slice(-1) === 's' ? referencesName.substring(0, referencesName.length-1) : referencesName}
````
with this 
```js
resourceName.replace(/s\s*$/, "")
```
the regexp is replacing the last 's' with ''